### ISSUE TITLE
LS changed IDE interface to add an additional "track ID" and it would be nice if sim channel makers provided.

### ISSUE THREAD

**Issue by brettviren** [LArSoft/larwirecell#34]
The current context for making `SimChannel` is this segment of the processing chain:

```
art::Event->

  (SimEnergyDeposit)->

[larwirecell] SimDepoSetSource->

  (IDepo)->

[WCT] Drifter->

  (IDepo)->

[larewirecell] DepoSetSimChannelSink->

  (SimChannel)->

art::Event
```

Probably obvious but ()'s marks data objects, []'s are packages, the rest are code objects.

Two changes are needed:

1) `SimDepoSetSource`

https://github.com/LArSoft/larwirecell/blob/develop/larwirecell/Components/SimDepoSetSource.cxx#L208

- ignore BOTH types of track ID.

- set the *loop index* as the `IDepo::id()`.

2) `DepoSetSimChannelSink`:

https://github.com/LArSoft/larwirecell/blob/develop/larwirecell/Components/DepoSetSimChannelSink.cxx#L189

- retrieve the vector of `SimEnergyDeposit` from `art::Event` - same as used  by `SimDepoSetSource`.

- use the `IDepo::id()` to index this vector.

- transfer **both** track IDs from `SimEnergyDeposit` to `SimChannel`.

Both of these changes should be made configurable in order to allow old behavior.  Eg, `SimDepoSetSource` and `DepoSetSimChannelSink` gain a parameter like `id_is_depo` which would be "true" in the new case and "false" in the old.

**Comment by brettviren** [comment]
Work on this is happening in https://github.com/brettviren/larwirecell/tree/feature/bv_trackid_torigin

Coding is done and now devising tests.

**Comment by brettviren** [comment]
Work merged in #35
